crates/iota/src/iota_commands.rs

```diff
@@ -727,7 +727,8 @@ async fn start(
                 remote_migration_snapshots,
                 delegator,
             )
-            .await?;
+            .await
+            .map_err(|_| anyhow!("Cannot run genesis with non-empty IOTA config directory: {}.\n\nIf you are trying to run a local network without persisting the data (so a new genesis that is randomly generated and will not be saved once the network is shut down), use --force-regenesis flag.\nIf you are trying to persist the network data and start from a new genesis, use iota genesis --help to see how to generate a new genesis.", config.display()))?;
```

Comment:
Can we format this pls?

Reply:
Done in https://github.com/iotaledger/iota/pull/4987/commits/0ca182229ff4238ab44a6b0708b16201b3212596